### PR TITLE
Fixes changeling thermal vision reverting.

### PR DESCRIPTION
**Pull request by EJDenton**
Reapplied the stop_sight_update check in life.dm. Tested successfully with changeling thermal vision.

### CODE REVIEW

**Comment by EJDenton** [comment]
!review

File: code/modules/mob/living/life.dm [hunk 1]
```diff
@@ -160,15 +160,16 @@
 		setEarDamage(-1, max(ear_deaf, 1))
 
 /mob/living/proc/update_sight()
-	set_sight(0)
-	if(stat == DEAD || (eyeobj && !eyeobj.living_eye))
-		update_dead_sight()
-	else
-		update_living_sight()
+	if(!stop_sight_update)
```
Comment (FluffyGhoster) [inline code/modules/mob/living/life.dm:163]: Flip this as a guard statement

if(stop_sight_update)
	return

[everything else]

**Comment by EJDenton** [comment]
!bugfix

File: code/modules/mob/living/life.dm (hunk 1 above)
Comment (EJDenton) [reply code/modules/mob/living/life.dm:163]: Change implemented.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**